public/scripts/power-user.js

```diff
@@ -1571,12 +1633,12 @@ async function loadPowerUserSettings(settings, data) {
     power_user.font_scale = Number(localStorage.getItem(storage_keys.font_scale) ?? 1);
     power_user.blur_strength = Number(localStorage.getItem(storage_keys.blur_strength) ?? 10);
 
-    if (power_user.chat_display === '') {
-        power_user.chat_display = chat_styles.DEFAULT;
+    if (power_user.chat_mode === '') {
+        power_user.chat_mode = chat_modes.DEFAULT;
```

Comment:
Shouldn't there be some kind of fallback that checks for the old `waifuMode` setting of user? Otherwise it's lost, and user has to set it again.